scripts/zones/Mamook/mobs/Hundredfaced_Hapool_Ja.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+-- Area: Mamook
+--  Mob: Hundredfaced Hapool Ja
+-----------------------------------
+require("scripts/globals/spell_data")
+require("scripts/globals/titles")
+require("scripts/globals/status")
+local ID = require("scripts/zones/Mamook/IDs")
+mixins = {require("scripts/mixins/job_special")}
+-----------------------------------
+local entity = {}
+
+entity.onMobFight = function(mob, target)
+    mob:addListener("MAGIC_USE", "SPAWN_CLONES", function(mobArg, target, spell, action)
```

Comment:
the listener should be added at spawn time instead of on every tick of onMobFight